controllers/virtualmachine/virtualmachine/virtualmachine_controller.go

```diff
@@ -144,7 +144,9 @@ func AddToManager(ctx *pkgctx.ControllerManagerContext, mgr manager.Manager) err
 		builder = builder.Watches(
 			&vmopv1.VirtualMachineGroup{},
 			handler.EnqueueRequestsFromMapFunc(
-				vmopv1util.GroupToVMsMapperFn(ctx, r.Client),
+				vmopv1util.GroupToMembersMapperFn(
+					ctx, r.Client, "VirtualMachine",
```

Comment:
Not part of this PR, thinking whether we should make `VirtualMachine ` a const in `pkg/util/vmopv1/vm.go`

Reply:
Agree. It's currently defined in vmgroup.go file in private. Will make it public and replace all uses in a separate PR.